apps/frontend/src/app/3_organisms/ZeroLocForm/components/AdjustCreditLine.tsx

```diff
@@ -126,23 +126,23 @@ export const AdjustCreditLine: FC<AdjustCreditLineProps> = ({
 
   const maxBorrowAmount = useMemo(() => {
     const collateral =
-      (collateralSize === 0 ? maxCollateralToDepositAmount : collateralSize) +
-      Number(existingCollateral);
+      collateralSize === 0
+        ? Number(existingCollateral)
+        : Number(existingCollateral) + collateralSize;
 
     const amount =
       (collateral * rbtcPrice) /
       (isRecoveryMode ? CRITICAL_COLLATERAL_RATIO : MINIMUM_COLLATERAL_RATIO);
 
     const originationFee = getOriginationFeeAmount(amount, borrowingRate);
 
-    return Math.max(amount - originationFee - Number(existingDebt), 0);
+    return Math.max(amount - Number(existingDebt) - originationFee, 0);
```

Comment:
@creed-victor it looks fine but it uses numbers instead of string. Will you take care of it in https://sovryn.atlassian.net/browse/SOV-1988 or do you want @pietro-maximoff to fix it in this task?

Reply:
yeah, will solve this on SOV-1988 @tiltom